resource-managers/kubernetes/integration-tests/src/test/scala/org/apache/spark/deploy/kubernetes/integrationtest/KubernetesV2Suite.scala

```diff
@@ -135,6 +135,19 @@ private[spark] class KubernetesV2Suite(testBackend: IntegrationTestBackend)
     runSparkPiAndVerifyCompletion(SparkLauncher.NO_RESOURCE)
   }
 
+  test("Use client key and client cert file when requesting executors") {
```

Comment:
is minikube configured at this point to require auth via the key/cert and CA cert files?  Theoretically this test could pass because nothing happens to the config, rather than that the config is set and used in the request to the apiserver which requires the auth config